docs/solidity-by-example.rst

```diff
@@ -86,7 +86,7 @@ of votes.
 
         // Give `voter` the right to vote on this ballot.
         // May only be called by `chairperson`.
-        function giveRightToVote(address voter) {
+        function GiveRightToVote(address voter) public {
```

Comment:
Regular functions should start with a lowercase letter. Does this need to to be capitalized? Concerning the other functions, I think it would be better to change local variables than function names, since function names are much more publicly visible.